src/router.js

```diff
@@ -23,7 +24,11 @@ const mustBeAuthenticated = (to, from, next) => {
   if (store.state.auth.authenticated) {
     return next();
   }
-  return next('/login');
+  const isReturningUser = store.dispatch('getReturningUser');
```

Comment:
The dispatch calls need to be updated to match the action rename
```suggestion
  const isReturningUser = store.dispatch('getIsReturningUser');
```